Sources/Transmute/transmute().swift

```diff
@@ -70,6 +70,15 @@ public func transmute(rootPackage: Package, externalPackages: [Package]) throws
 
     let modules = recursiveDependencies(packages.flatMap{ map[$0] ?? [] })
     let externalModules = recursiveDependencies(externalPackages.flatMap{ map[$0] ?? [] })
+    
+    let (testProducts, nonTestProducts) = products.partition{
```

Comment:
I don't think this needs to be in transmute, any component anywhere can split products into tests and non-tests as that information is carried in the Product type.